roles/node/files/testnet/blocked_ips.txt

```diff
@@ -0,0 +1 @@
+# empty file
```

Comment:
remove the comment, I just added it to help you understand, this txt file, comment wont really work here